### DISCUSSION TITLE
How to use a logger in an asyncio background task

### DISCUSSION THREAD

**Discussion by ElieTaillard** [Rapptz/discord.py#9654] [Q&A] [answered]
Hello, I would like to know how to use a logger (from the logging module) in this example:
https://github.com/Rapptz/discord.py/blob/master/examples/background_task_asyncio.py
I tried on my side but I get duplicate logs:
![image](https://github.com/Rapptz/discord.py/assets/54487782/ca12b637-4e65-44de-bd03-67d3072af051)
Do I need to pass anything in `client.run('token')` for it to work properly?
https://github.com/Rapptz/discord.py/blob/master/examples/background_task_asyncio.py#L28C1-L28C20

**Comment by LostLuma** [comment]
Hello, if you're setting up log related things yourself you'll indeed want to disable it using the params passed to `Client.run`, see the [relevant documentation here](https://discordpy.readthedocs.io/en/stable/api.html#discord.Client.run).

**Reply by ElieTaillard** [reply]
Can I override the logger ('discord') ?

**Reply by LostLuma** [reply]
What do you mean by override?

**Reply by Motzumoto** [reply]
Maybe by doing something like this?:
```python
logging.getLogger("discord").setLevel(logging.Whatever_level_you_want)
```

**Comment by ElieTaillard** [comment, marked as answer]
By following this [guide](https://discordpy.readthedocs.io/en/stable/logging.html), I realized that I could disable the log configuration.
By using the run function like this : `client.run(token, log_handler=None)`, the logs no longer seem to be duplicated. My problem is now solved, but I appreciate your efforts to provide me with a solution
